src/screens/SendConfirmationSheet.tsx

```diff
@@ -289,6 +299,73 @@ export default function SendConfirmationSheet() {
         ]
   );
 
+  useEffect(() => {
+    if (isENS) {
+      const promises = checkboxes
+        .filter(option => option.checked)
+        .map(option => {
+          switch (option.id) {
+            case 'clear-records':
+              const emptyRecords = Object.keys(
+                ensProfile?.data?.records || {}
+              ).reduce((records, recordKey) => {
+                return {
+                  ...records,
+                  [recordKey]: '',
+                };
+              }, {});
+              return estimateENSSetRecordsGasLimit({
+                name: asset?.name,
+                records: emptyRecords,
+              });
+            case 'set-address':
+              return estimateENSSetNameGasLimit({
```

Comment:
Just a note, but we don't actually call `setName` for "set-address", it's actually batched up with the set records stuff. Check out [here](https://github.com/rainbow-me/rainbow/pull/3389/files#diff-00be62de6f93f2c78dd5e9401fd2e6fb92d1f534b7b60593e012b7bb32779c0dR142-R147) and [here](https://github.com/rainbow-me/rainbow/pull/3389/files#diff-00be62de6f93f2c78dd5e9401fd2e6fb92d1f534b7b60593e012b7bb32779c0dR166-R170).

Reply:
cool ty